audio_service/test/isolates_test.dart

```diff
@@ -4,241 +4,339 @@ import 'package:audio_service/audio_service.dart';
-        await testMethod(
-            'removeQueueItem', () => proxy.removeQueueItem(Data.mediaItem));
-        await testMethod('removeQueueItemAt', () => proxy.removeQueueItemAt(0));
-        await testMethod('skipToNext', () => proxy.skipToNext());
-        await testMethod('skipToPrevious', () => proxy.skipToPrevious());
-        await testMethod('fastForward', () => proxy.fastForward());
-        await testMethod('rewind', () => proxy.rewind());
-        await testMethod('skipToQueueItem', () => proxy.skipToQueueItem(0));
-        await testMethod('seek', () => proxy.seek(Duration.zero));
-        await testMethod(
-            'setRating',
-            () => proxy.setRating(
-                const Rating.newHeartRating(true), <String, dynamic>{}));
-        await testMethod(
-            'setCaptioningEnabled', () => proxy.setCaptioningEnabled(true));
-        await testMethod('setRepeatMode',
-            () => proxy.setRepeatMode(AudioServiceRepeatMode.all));
-        await testMethod('setShuffleMode',
-            () => proxy.setShuffleMode(AudioServiceShuffleMode.all));
-        await testMethod('seekBackward', () => proxy.seekBackward(true));
-        await testMethod('seekForward', () => proxy.seekForward(true));
-        await testMethod('setSpeed', () => proxy.setSpeed(1.5));
-        await testMethod('onTaskRemoved', () => proxy.onTaskRemoved());
-        await testMethod(
-            'onNotificationDeleted', () => proxy.onNotificationDeleted());
-        await testMethod('getChildren',
-            () => proxy.getChildren(Data.mediaId, <String, dynamic>{}),
-            expectedResult: Data.mediaItems);
-        await testMethod('subscribeToChildren',
-            () => proxy.subscribeToChildren(Data.mediaId));
-        await testMethod('getMediaItem', () => proxy.getMediaItem(Data.mediaId),
-            expectedResult: Data.mediaItem);
-        await testMethod(
-            'search', () => proxy.search(Data.query, <String, dynamic>{}),
-            expectedResult: Data.mediaItems);
-        await testMethod(
-            'androidSetRemoteVolume', () => proxy.androidSetRemoteVolume(3));
-        await testMethod(
-            'androidAdjustRemoteVolume',
-            () =>
-                proxy.androidAdjustRemoteVolume(AndroidVolumeDirection.raise));
-        await testMethod('customAction',
-            () => proxy.customAction('echo', <String, dynamic>{'arg': 'foo'}),
-            expectedResult: 'foo');
-        await testMethod('customAction',
-            () => proxy.customAction('echo', <String, dynamic>{'arg': 'bar'}),
-            expectedResult: 'bar');
-      });
-
-      test('stream values $isolateLabel', () async {
-        Future<void> testStream<T>(
-            String name, ValueStream<T> stream, T value) async {
-          await proxy.add(name, value);
-          expect(stream.nvalue, value);
-        }
+          expect(result, expectedResult);
```

Comment:
Always expect the method to return some value. This ensures we don't forget to test the returned method value.